components/basecomponents/MemoryGame/MemoryGameThemeLoader.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from 'react';
+import MemoryGameLoading from 'components/basecomponents/MemoryGame/MemoryGameLoading';
+import styles from './MemoryGameThemeLoader.module.css';
+import Image from 'next/image';
+
+const DefaultTheme = React.lazy(() => import('./Themes/MemoryGameDefaultTheme'));
+const TaleTheme = React.lazy(() => import('./Themes/MemoryGameTaleTheme'));
+
+export enum Theme {
```

Comment:
Why keep Theme and themeImages and Theme components separately? It just complicates your types and mark-up. Don't you want to do something like 

```
type CartoonTheme = {
  id: string;
  image: string;
  component: React.LazyExoticComponent<() => JSX.Element>;
}

const themes: CartoonTheme[] = [
  {
    id: "default",
    image: '/kids/memory-game/card_back_movapp.png',
    component: DefaultTheme,
  }
]
```